### DISCUSSION TITLE
Want to add a new logo and use it

### DISCUSSION THREAD

**Discussion by dushmanta05** [badges/shields#9135] [Q&A] [answered]
Hi everyone,

I'm working on my GitHub profile repository and I want to add a profile link logo for a website called peerlist.io using the same format as the LinkedIn logo in my README file, like this:
(https://img.shields.io/badge/LinkedIn-%230077B5.svg?logo=linkedin&logoColor=white)

I have the SVG file for the peerlist logo, and I'd like to know how to add this logo to the project so that using the "logo = peerlist" parameter will work. Can someone please explain how I can do this?

Thank you in advance!

**Comment by calebcartwright** [comment, marked as answer]
https://github.com/badges/shields/blob/master/doc/logos.md

**Reply by dushmanta05** [reply]
Thank you

**Reply by calebcartwright** [reply]
yw!

**Comment by krishna9786** [comment]
Upload the peerlist.io logo SVG file to your GitHub repository.

Generate the badge URL using Shields.io service in the following format:

bash
[![Peerlist](https://img.shields.io/badge/Peerlist.io-<COLOR>.svg?logo=data:image/svg+xml;base64,<BASE64_ENCODED_SVG>)](https://www.peerlist.io/)

Replace <COLOR> with the desired badge background color (e.g., "blue," "red," etc.).
Obtain the base64 encoded string of the peerlist.io SVG logo file and replace <BASE64_ENCODED_SVG> in the badge URL.
Commit the changes to your README file, and the badge with the peerlist.io logo should appear in your GitHub profile repository.
Remember to adhere to the logo usage rights and guidelines provided by peerlist.io.
